packages/cli/src/lib/missing-routes.ts

```diff
@@ -30,12 +30,16 @@ const REQUIRED_ROUTES = [
 
   'account',
   'account/login',
-  'account/register',
   // 'account/addresses',
   // 'account/orders',
   'account/orders/:orderId',
-  'account/reset/:id/:token',
-  'account/activate/:id/:token',
+
+  // -- These were removed when migrating to CAAPI:
+  // 'account/register',
+  // 'account/reset/:id/:token',
+  // 'account/activate/:id/:token',
+  // -- Added for CAAPI:
+  'authorize',
```

Comment:
The only way to know which API we are using is by parsing `server.ts`. Not impossible but not sure it's the best option.

In any case, I think this might be unrelated to what API we use and more on the side of being in sync with Liquid, in case some online store redirect hits one of these routes 🤔  -- e.g. imagine a link in an email to `acount/activate/...` or similar, but not sure if that's even possible.